src/keys.test.ts

```diff
@@ -0,0 +1,11 @@
+import { keys } from './keys';
+
+describe('Test for values', () => {
+  it('should return values of array', () => {
+    expect(keys(['x', 'y', 'z'])).toEqual(['0', '1', '2']);
+  });
+
+  it('should return values of object', () => {
+    expect(keys({ a: 'x', b: 'y', c: 'z' })).toEqual(['a', 'b', 'c']);
+  });
+});
```

Comment:
this should be `test for keys` I guess, and `should return keys...`

looks like a copy-paste error from the values tests :)